### PR TITLE
Keep navbar's "View on GitHub" button up-to-date

### PR DESCRIPTION
**Pull request by mbland**
cc: @maya

### CODE REVIEW

**Comment by maya** [comment]
@mbland can you resolve merge conflicts?

**Comment by mbland** [comment]
@maya Done!

File: pages/getting-started.html [hunk 1]
```diff
@@ -55,8 +55,8 @@ <h3 class="usa-heading">For designers</h3>
...
-  <li><strong>Illustrator, EPS, Sketch, and Omnigraffle</strong> art files of each component and pattern on this site are available at: <a href="https://github.com/18F/web-design-standards-assets">github.com/18F/web-design-standards-assets</a></li>
-  <li><strong>.AI and .ASE color swatches</strong> are available at:</strong> <a href="https://github.com/18F/web-design-standards-assets/tree/master/Colors">github.com/18F/web-design-standards-assets/tree/master/Colors</a></li>
-  <li><strong>.TTF files</strong> of the font families and weights we recommend are available at: <a href="https://github.com/18F/web-design-standards-assets/">github.com/18F/web-design-standards-assets/</a> (Note: all fonts used in the Web Design Standards are free, open source typefaces also available online).</li>
+  <li><strong>Illustrator, EPS, Sketch, and Omnigraffle</strong> art files of each component and pattern on this site are available at: <a href="{{ site.repos[1].url }}">{{ site.repos[1].url }}</a></li>
+  <li><strong>.AI and .ASE color swatches</strong> are available at:</strong> <a href="{{ site.repos[1].url }}/tree/master/Colors">{{ site.repos[1].url }}/tree/master/Colors</a></li>
```
Comment (maya) [inline pages/getting-started.html]: @mbland this link doesn't work.

File: pages/getting-started.html [hunk 2]
```diff
@@ -55,8 +55,8 @@ <h3 class="usa-heading">For designers</h3>
...
 <p>All of these designs are also available in a variety of design file formats for download:</p>
 <ul class="usa-content-list">
-  <li><strong>Illustrator, EPS, Sketch, and Omnigraffle</strong> art files of each component and pattern on this site are available at: <a href="https://github.com/18F/web-design-standards-assets">github.com/18F/web-design-standards-assets</a></li>
-  <li><strong>.AI and .ASE color swatches</strong> are available at:</strong> <a href="https://github.com/18F/web-design-standards-assets/tree/master/Colors">github.com/18F/web-design-standards-assets/tree/master/Colors</a></li>
-  <li><strong>.TTF files</strong> of the font families and weights we recommend are available at: <a href="https://github.com/18F/web-design-standards-assets/">github.com/18F/web-design-standards-assets/</a> (Note: all fonts used in the Web Design Standards are free, open source typefaces also available online).</li>
+  <li><strong>Illustrator, EPS, Sketch, and Omnigraffle</strong> art files of each component and pattern on this site are available at: <a href="{{ site.repos[1].url }}">{{ site.repos[1].url }}</a></li>
```
Comment (maya) [inline pages/getting-started.html]: @mbland there's no link here.

File: pages/getting-started.html (hunk 1 above)
Comment (maya) [inline pages/getting-started.html]: Needed to restart server, bc _config.yml was modified. This works.

File: pages/getting-started.html (hunk 2 above)
Comment (maya) [inline pages/getting-started.html]: Needed to restart server, bc _config.yml was modified. This works.

**Comment by mbland** [comment]
Just for the record, confirmed that @maya needed to restart her local Jekyll server, since the aforementioned link as just added to `_config.yml`. :-)

**Pull request merged**